src/shopify/theme_client.go

```diff
@@ -326,6 +341,8 @@ func (c Client) DeleteAsset(asset Asset) error {
 		return ErrNotPartOfTheme
 	} else if resp.StatusCode == 406 {
 		return ErrMissingAssetName
+	} else if resp.StatusCode == 409 {
```

Comment:
> **409 Conflict**
> Indicates that the request could not be processed because of conflict in the current state of the resource, such as an edit conflict between multiple simultaneous updates.

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes